casts/cards/src/UIMediaCard/QuickView.tsx

```diff
@@ -1,16 +1,16 @@
 import * as React from 'react';
-import { View, StyleSheet } from 'react-native';
-import { UIImage } from '@tonlabs/uikit.media';
+// import { View } from 'react-native';
+// import { UIImage } from '@tonlabs/uikit.media';
 
 import type { QuickViewProps } from './types';
+import { CollectionSlide } from '../UICollectionCard/CollectionSlide';
 
-export const QuickView: React.FC<QuickViewProps> = ({ source, style, contentType }: QuickViewProps) => {
-    if (contentType !== 'Image' ||!source) {
+export const QuickView: React.FC<QuickViewProps> = ({ content, style }: QuickViewProps) => {
```

Comment:
Do you think we can drop this component? too much of a fragmentation if you ask me :)